spec/unit/inventory_v4_spec.rb

```diff
@@ -0,0 +1,47 @@
+require_relative '../spec_helper'
+
+module Hawkular::Inventory::RSpecUnit
```

Comment:
Why is this module necessary?

Reply:
Not really sure :) I saw other inventory specs do the same.
If you think is safe to be removed, then I can do that.

Reply:
If it is not, then it should be changed before moving forward. Even if that's correct for some reason, specs should not be contained in modules because that would make the setup different from the usage because of namespacing.